frontend/src/pages/VerifyEmail.jsx

```
@@ -0,0 +1,116 @@
+            >
+              {loading ? "Verifying..." : "Verify OTP"}
+            </button>
+          </form>
+
+          <p className="text-center text-sm mt-6">
+            Didn't receive the OTP?{" "}
+            <button
+              onClick={handleResend}
+              className="text-blue-600 hover:underline font-medium"
+              disabled={resendLoading}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)